archetype_flask_connexion/database/db_utils.py

```diff
@@ -0,0 +1,65 @@
+"""Utility functions for MongoDB document insertion, updates and retrieval."""
+
+from typing import (Any, Mapping, Optional, Union)
+
+from bson.objectid import ObjectId
+from pymongo.collection import ReturnDocument
+from pymongo import collection as Collection
+
+
+def find_one_latest(collection: Collection) -> Optional[Mapping[Any, Any]]:
+    """Returns newest/latest object, stripped of the object id, or None if no
+    object exists: collection.
+    """
+    try:
+        return collection.find(
+            {},
+            {'_id': False}
+        ).sort([('_id', -1)]).limit(1).next()
+    except StopIteration:
+        return None
+
+
+def find_id_latest(collection: Collection) -> Optional[ObjectId]:
```

Comment:
If I see this correctly, this function copies the exact same code as the function above with a simple addition of `['_id']`. I think it would be better to just simply replace the function body with:

```py
return find_one_latest(collection=collection)['_id']
```

Reply:
But
`        return collection.find(
            {},
            {'_id': False}
        ).sort([('_id', -1)]).limit(1).next()`

in `find_one_latest` returns object without `_id` attribute, so `return find_one_latest(collection=collection)['_id']` will give key error